gazebo_ros/launch/gzserver.launch.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Launch a Gazebo server with command line arguments."""
+
+from os import environ
+
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.actions import ExecuteProcess
+from launch.substitutions import LaunchConfiguration
+from launch.substitutions import PythonExpression
+
+from scripts import GazeboRosPaths
+
+
+def generate_launch_description():
+    model, plugin, media = GazeboRosPaths.get_paths()
+
+    if 'GAZEBO_MODEL_PATH' in environ:
+        model += ':'+environ['GAZEBO_MODEL_PATH']
+    if 'GAZEBO_PLUGIN_PATH' in environ:
+        plugin += ':'+environ['GAZEBO_PLUGIN_PATH']
+    if 'GAZEBO_RESOURCE_PATH' in environ:
+        media += ':'+environ['GAZEBO_RESOURCE_PATH']
+
+    env = {'GAZEBO_MODEL_PATH': model,
+           'GAZEBO_PLUGIN_PATH': plugin,
+           'GAZEBO_RESOURCE_PATH': media}
+
+    return LaunchDescription([
+        DeclareLaunchArgument('world', default_value='',
+                              description='Specify world file name'),
+        DeclareLaunchArgument('version', default_value='false',
+                              description='Set "true" to output version information.'),
+        DeclareLaunchArgument('verbose', default_value='false',
+                              description='Set "true" to increase messages written to terminal.'),
+        DeclareLaunchArgument('help', default_value='false',
+                              description='Set "true" to produce gzserver help message.'),
+        DeclareLaunchArgument('pause', default_value='false',
```

Comment:
Note, in ROS 1 the launch argument name is "pause**d**". I'm not sure why it changed in ROS 2, but maybe we can document this difference somewhere on a migration page.

Reply:
I'm referring to the [empty_world.launch](https://github.com/ros-simulation/gazebo_ros_pkgs/blob/0606994ee2cd42afed75b32c494cacc6fe8adf16/gazebo_ros/launch/empty_world.launch) file, but I guess this isn't really an exact equivalent to `gzserver.launch.py`, they just happen to both offer launch args for pausing.